category/interfaces/graphql/resolvers.go

```diff
@@ -3,23 +3,33 @@ package graphql
 import (
 	"context"
 	"flamingo.me/flamingo-commerce/v3/category/domain"
-	"flamingo.me/flamingo-commerce/v3/category/interfaces/controller"
+	graphqlDto "flamingo.me/flamingo-commerce/v3/category/interfaces/graphql/dto"
 	productApplication "flamingo.me/flamingo-commerce/v3/product/application"
-	searchApplication "flamingo.me/flamingo-commerce/v3/search/application"
-	searchDomain "flamingo.me/flamingo-commerce/v3/search/domain"
-	searchGraphQlDto "flamingo.me/flamingo-commerce/v3/search/interfaces/graphql/dto"
+	productDomain "flamingo.me/flamingo-commerce/v3/product/domain"
+	"flamingo.me/flamingo-commerce/v3/search/interfaces/graphql/dto"
 )
 
 // CommerceCategoryQueryResolver resolves graphql category queries
 type CommerceCategoryQueryResolver struct {
 	categoryService      domain.CategoryService
-	productSearchService *productApplication.ProductSearchService
```

Comment:
Whats the reason for not using the existing application Service?

Reply:
We tried to align to the product search query resolver. Anyway i've updated both resolvers, so they both use product application service. Review updated.